src/display.rs

```diff
@@ -114,9 +36,14 @@ fn inner_display_grid(
 ) -> String {
     let mut output = String::new();
 
+    let padding_rules = get_padding_rules(&metas, flags);
     let mut grid = Grid::new(GridOptions {
         filling: Filling::Spaces(2),
```

Comment:
This seems to be how it is handled everywhere else. I think we should go with this too.

```suggestion
         filling: if flags.layout == Layout::OneLine {
            Filling::Spaces(1)
         } else {
            Filling::Spaces(2)
         },
```